.github/workflows/ci.yml

```diff
@@ -412,7 +431,13 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install wheel
-          pip install --no-cache-dir .
+        
+          HUBBLE_VERSION=$(curl -L -s "https://pypi.org/pypi/jina-hubble-sdk/json" |  jq  -r '.releases | keys | .[]| select(startswith("0."))'  | sort -V | tail -1)
```

Comment:
what is this doing?

Reply:
getting the latest version on pypi. I copy paste from the executor CI